apache/zeppelin
# [ZEPPELIN-1796] Add 'copy to clipboard' next to the paragraph id

### What is this PR for?
There are lots of use cases that need to know each `Paragraph Id` in Zeppelin. 
e.g. [run paragraph using rest api](https://zeppelin.apache.org/docs/0.7.0-SNAPSHOT/rest-api/rest-notebook.html#run-a-paragraph-asynchronously) / [`z.angularBind`](https://zeppelin.apache.org/docs/0.7.0-SNAPSHOT/displaysystem/front-end-angular.html#basic-usage) / [export paragraphs using iframe](https://zeppelin.apache.org/docs/0.7.0-SNAPSHOT/manual/publish.html) ...

So I added `Copy to clipboard` btn next the each `Paragraph Id` under paragraph setting menu using [angular-zeroclipboard](https://github.com/lisposter/angular-zeroclipboard). 
It'll be useful even if it's small feature :)

### What type of PR is it?
Improvement | Feature 

### What is the Jira issue?
[ZEPPELIN-1796](https://issues.apache.org/jira/browse/ZEPPELIN-1796)

### How should this be tested?
1. `npm run build` under `zeppelin-web/`
2. `npm run start` to start dev mode and browse `http://localhost:9000`
3. click gear icon placed top of the each paragraph -> hover the mouse on the paragraph id 
4. click it and paste the paragraph id to anywhere to check it's copied well or not

### Screenshots (if appropriate)
![copy_pid](https://cloud.githubusercontent.com/assets/10060731/21213283/9fb53710-c2d6-11e6-981d-58174da76196.gif)


### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no 
* Does this needs documentation? no
